contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -179,6 +181,11 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
     return s_config.maxConsumersPerSubscription;
   }
 
+  // Used within FunctionsSubscriptions.sol
+  function _getSubscriptionDepositDetails() internal view override returns (uint16, uint72, address) {
```

Comment:
Can we maybe use getConfig() instead of adding a new method?

Reply:
Nope, `FunctionsSubscriptions` has no concept of the Router, so wouldn't know which contract to call `getConfig` on. We still would need a virtual function that gets overridden